fh-ios-sdk/FHConfig.m

```diff
@@ -21,13 +21,18 @@ - (instancetype)init {
     self = [super init];
     if (self) {
         NSString *path =
-            [[NSBundle bundleForClass:[self class]] pathForResource:@"fhconfig" ofType:@"plist"];
+            [[NSBundle mainBundle] pathForResource:@"fhconfig" ofType:@"plist"];
         if (path) {
             self.properties = [NSMutableDictionary dictionaryWithContentsOfFile:path];
         } else {
-            @throw([NSException exceptionWithName:@"fhconfigException"
-                                           reason:@"fhconfig.plist was not located"
-                                         userInfo:nil]);
+            path = [[NSBundle bundleForClass:[self class]] pathForResource:@"fhconfig" ofType:@"plist"];
+            if (path) {
+                self.properties = [NSMutableDictionary dictionaryWithContentsOfFile:path];
+            } else {
+                @throw([NSException exceptionWithName:@"fhconfigException"
+                                               reason:@"fhconfig.plist was not located"
```

Comment:
"Couldn't find required configuration file fhconfig.plist in the main application bundle."